### ISSUE TITLE
Footer headings should be h2 main h1 is missing

### ISSUE THREAD

**Issue by femmebot** [18F/nsf-sbir#698]
## Actual Behavior
Programmatically identified heading levels do not match the visual outline level. 

## Steps to reproduce the behavior
- [x] View the page outline. Ensure the headings are logical. One h1 per page, sub headings don't skip a level, elements of similar import are on the same heading level

see: 508 compliance report [Portfolio details N° 90](https://docs.google.com/spreadsheets/d/1yWVeVHPJtoqsCTEwnYalqDTSXNkr5E_Bn20msVmkP-8/edit#gid=0)
